src/Components/DataHistoryExplorerModal/DataHistoryExplorerModal.styles.ts

```diff
@@ -0,0 +1,21 @@
+import {
+    IDataHistoryExplorerModalStyleProps,
+    IDataHistoryExplorerModalStyles
+} from './DataHistoryExplorerModal.types';
+
+export const classPrefix = 'cb-datahistoryexplorermodal';
+const classNames = {
+    root: `${classPrefix}-root`,
+    contentStack: `${classPrefix}-content-stack`
+};
+export const getStyles = (
+    _props: IDataHistoryExplorerModalStyleProps
+): IDataHistoryExplorerModalStyles => {
+    return {
+        root: [classNames.root],
+        contentStack: [classNames.contentStack, { height: '100%' }],
+        subComponentStyles: {
+            modal: { titleContainer: { i: { paddingTop: 0 } } }
```

Comment:
there is a property name called "i"?

Reply:
this is for aligning the text with the icon in the modal title:
<img width="841" alt="image" src="https://user-images.githubusercontent.com/45217314/210455815-33e59eb7-3737-401c-8b16-af4a578b857e.png">

Reply:
gotchaaa.... Could we just put those in a horizontal stack instead?